### ISSUE TITLE
Tabs on Front Page

### ISSUE THREAD

**Issue by lydia1wang** [writingatlas/feedback#2]
<img width="1134" alt="Screenshot 2023-12-18 at 11 53 31 AM" src="https://github.com/writingatlas/feedback/assets/153849658/b6d9d21e-a7e1-40fe-9d7d-c157b8b58fbc">

On the main page, many of the tabs lead to the same "discovering stories" page, causing redundancy. Some examples of tabs leading to the same pages include: "for readers: find new stories", "for readers: rediscovering reading", "for writers: get discovered", "for writers: build your audience", etc.

Additionally, "for industry: be in the know" does not work.

**Comment by srhm-ca** [comment]
Yep! The hope is to slowly build out site functionality to give each button a unique destination. It's definitely a problem for now, yes.
